packages/state_queue/test/state_queue/state_queue_test.dart

```diff
@@ -7,13 +7,13 @@ import 'package:test/test.dart';
 class _TestBloc extends StateQueue<int> {
   _TestBloc() : super(0);
 
-  void setState(int n) {
+  void setState(int n) async {
```

Comment:
Why `async` when they just return a `void`? Shouldn't change anything here.

Reply:
(Just makes me think that we should copy over the analyze configuration)

Reply:
Yeah let's do it after #20